cache/fast_lru_cache.cc

```diff
@@ -21,20 +21,13 @@
 #include "port/lang.h"
 #include "util/distributed_mutex.h"
 #include "util/hash.h"
+#include "util/math.h"
 #include "util/random.h"
 
 namespace ROCKSDB_NAMESPACE {
 
 namespace fast_lru_cache {
 
```

Comment:
BinaryMod moved to util/math.h. It's now also used by the clock cache.

Reply:
It is common to save a bit mask `(uint32_t{1} << bits) - 1` and use `&` on that mask instead. It looks here like we never really need the number of bits, just the mask or mask + 1.